pkg/udev/common.go

```diff
@@ -121,6 +121,7 @@ func (device *UdevDevice) GetUid() string {
 	localDiskModels := make([]string, 0)
 	localDiskModels = append(localDiskModels, "EphemeralDisk")
 	localDiskModels = append(localDiskModels, "Virtual_disk")
+	localDiskModels = append(localDiskModels, "QEMU_HARDDISK")
```

Comment:
This will help to identify KVM disks when the disk is not having a unique serial number right?

Reply:
Exactly.